app/controllers/moderations_controller.rb

```diff
@@ -37,15 +37,6 @@ def comment
 
   def actions_panel
     load_article
-    tag_mod_tag_ids = @tag_moderator_tags.ids
```

Comment:
I moved part of this logic in the controller to the `article_policy` so that we can perform the policy checks there.